lib/player.js

```diff
@@ -5640,6 +5651,19 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return info;
   }
 
+  /**
+   * Get latency in milliseconds between the live edge and what's currently
+   * playing.
+   *
+   * @return {number}
+   */
+  getLiveLatency() {
+    const element = this.video_;
+    const now = this.getPresentationStartTimeAsDate().valueOf() +
```

Comment:
I know it's technically the same but a bit easier to read
```suggestion
    const now = this.getPresentationStartTimeAsDate().getTime() +
```